test/checkpointctl.bats

```diff
@@ -589,3 +589,22 @@ function teardown() {
 	[ "$status" -eq 1 ]
 	[[ ${lines[0]} == *"no process with PID 9999"* ]]
 }
+
+@test "Run checkpointctl list with no checkpoints" {
+    checkpointctl list
+    [ "$status" -eq 0 ]
+    [[ ${lines[1]} == *"No checkpoints found in this path."* ]]
+}
+
+@test "Run checkpointctl list with checkpoints in tar files" {
+    mkdir -p "$TEST_TMP_DIR1"/checkpoint-1
+    mkdir -p "$TEST_TMP_DIR1"/checkpoint-2
+
+    tar cf "$TEST_TMP_DIR2"/checkpoint-pod-name_namespace1-container-name-2023-10-12T11:02:56+01:00.tar -C "$TEST_TMP_DIR1" checkpoint-1
+    tar cf "$TEST_TMP_DIR2"/checkpoint-counter-tl58j_default-counter-2023-10-12T11:02:56+01:00.tar -C "$TEST_TMP_DIR1" checkpoint-2
+
+    checkpointctl list -p "$TEST_TMP_DIR2"
+    [ "$status" -eq 0 ]
+    [[ "${lines[8]}" =~ "checkpoint-counter-tl58j_default-counter-2023-10-12T11:02:56+01:00.tar" ]]
+    [[ "${lines[6]}" =~ "checkpoint-pod-name_namespace1-container-name-2023-10-12T11:02:56+01:00.tar" ]]
```

Comment:
It would be good to verify that namespace, pod, and container are correctly parsed in the output table.

Reply:
currently I added assertion for other fields as same as the chekpoint name assertion in line 4 and 6. I think it could be done in a better way if we can parse those field's value in an array and then asserts them one by one using index value. is that possible? I am really not aware of that. anyway I will check it and if that possible then we can do in the better way.